src/Cli/Microsoft.TemplateEngine.Cli/Commands/create/TemplateCommand.cs

```diff
@@ -10,10 +10,11 @@
 using Microsoft.TemplateEngine.Edge;
 using Microsoft.TemplateEngine.Edge.Settings;
 using Microsoft.TemplateEngine.Utils;
+using Command = System.CommandLine.Command;
```

Comment:
I was getting the following error:

```log
'Command' is an ambiguous reference between 'System.CommandLine.Command' and 'Microsoft.DotNet.Cli.Utils.Command'
```

and decided to make it explicit with a `using` rather than using fully qualified name everywhere